test/index.spec.js

```diff
@@ -111,11 +118,21 @@ function runTestWithConfig (testConfig, initialStoreState, label) {
 
     describe('Check non redux-undo actions', () => {
       it('should not record unwanted actions', () => {
-        if (testConfig.FOR_TEST_ONLY_excludedActions && testConfig.FOR_TEST_ONLY_excludedActions[0]) {
+        if (testConfig.FOR_TEST_ONLY_excludedActions) {
+          // don't record this action in history
           let decrementedState = mockUndoableReducer(mockInitialState, { type: testConfig.FOR_TEST_ONLY_excludedActions[0] })
           expect(decrementedState.past).to.deep.equal(mockInitialState.past)
           expect(decrementedState.future).to.deep.equal(mockInitialState.future)
         }
+
+        if (testConfig.FOR_TEST_ONLY_includeAction) {
+          // only record this action in history
+          let tmpState = mockUndoableReducer(mockInitialState, { type: testConfig.FOR_TEST_ONLY_includeAction[0] })
+          let expected = { ...tmpState, present: tmpState.present + 1 }
+          // and not this one...
+          tmpState = mockUndoableReducer(tmpState, { type: 'INCREMENT' })
+          expect(tmpState).to.deep.equal(expected)
+        }
```

Comment:
I think ` if (testConfig.FOR_TEST_ONLY_excludedActions && testConfig.FOR_TEST_ONLY_excludedActions[0]) ` is not really necessary. We know that if it's set that it's an array and contains 1 item.